### PR TITLE
fix: sync btn claim

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- fix: sync status 2 buttons of the same campaign when clicking claim reward.
- fix: can not move to next page of lucky winner
- update ux: hide progressbar if number of trade <= 1 
image

### CODE REVIEW

File: src/state/campaigns/hooks.ts [hunk 1]
```diff
@@ -29,6 +31,20 @@ export function useSelectedCampaignLeaderboardPageNumberManager() {
...
+  const dispatch = useDispatch()
+
+  const setPage = useCallback(
+    (newPageNumber: number) => {
+      dispatch(setSelectedCampaignLuckyWinnersPageNumber(newPageNumber))
+    },
+    [dispatch],
+  )
+
+  return useMemo(() => [page, setPage], [page, setPage])
```
Comment (hungdoansy) [inline src/state/campaigns/hooks.ts:45]: I think useMemo is not quite needed here. we can remove it

**Review by hungdoansy — APPROVED** [review]
LGTM

File: src/state/campaigns/hooks.ts (hunk 1 above)
Comment (namgold) [reply src/state/campaigns/hooks.ts:45]: It does need indeed. `return [page, setPage]` creates new instance of array regards of value of `page` and `setPage` are same as before

Comment (hungdoansy) [reply src/state/campaigns/hooks.ts:45]: But why do you need to memo them? I see he spreaded them eventually 
```js
const [currentPageLuckywinner, setCurrentPageLuckywinner] = useSelectedCampaignLuckyWinnerPageNumber()
```

Comment (nguyenhoaidanh) [reply src/state/campaigns/hooks.ts]: ah, after looking again,  I also see that it was not necessary. I just copied and edited the name so I didn't notice.

sorry. :D
I updated.

Comment (hungdoansy) [reply src/state/campaigns/hooks.ts]: 🤝

Comment (nguyenhoaidanh) [reply src/state/campaigns/hooks.ts]: image

Comment (namgold) [reply src/state/campaigns/hooks.ts]: image

**Review by hungdoansy — APPROVED** [review]

**Review by namgold — APPROVED** [review]

**Pull request merged**